lang/python/python3-networkx/Makefile

```
@@ -0,0 +1,39 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-networkx
+PKG_VERSION:=2.5
+PKG_RELEASE:=1
+
+PYPI_NAME:=networkx
+PKG_HASH:=7978955423fbc9639c10498878be59caf99b44dc304c2286162fd24b458c1602
+
+PKG_LICENSE:=BSD-3-clause
+PKG_LICENSE_FILES:=LICENSE.txt
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-networkx
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python package for creating and manipulating graphs and networks
```

```suggestion
  TITLE:=Creating and manipulating graphs and networks
```

Not a big issue, but the title is probably a bit too long for menuconfig.

---

Time to switch to `nconfig` :)

---

Fixed.

---

@aparcar, CI is failing after this superficial change. Is the CI system okay?